src/lucky.cr

```diff
@@ -15,6 +15,7 @@ require "./lucky/exceptions"
 require "./lucky/response"
 require "./lucky/cookies/*"
 require "./lucky/secure_headers/*"
+require "./lucky/route_helper"
```

Comment:
So why does this one specifically need to be required where there's a ton of other files that do not? Is this because of an order where the RouteHelper is called before some others?

Reply:
Yes. Some file in `./lucky/*` that is before route_helper uses `: RouteHelper` explicitly. Both, `redirectable.cr` and `tags/form_helpers.cr` match that. Not sure if one or both were the culprit.